src/tests/assert.ts

```diff
@@ -20,17 +20,16 @@ export module Assert {
 
 		// Assert tab index ordering
 		for (let i = 1; i < elements.length; i++) {
-			ok(elements[i].elem.tabIndex > elements[i - 1].elem.tabIndex,
-				"Element " + elements[i].name + " should have a greater tabIndex than element " + elements[i - 1].name);
-		}
+				ok(elements[i].elem.tabIndex > elements[i - 1].elem.tabIndex, "Element " + elements[i].name + " whose tabIndex is" + elements[i].elem.tabIndex + " should have a greater tabIndex than element " + elements[i - 1].name + " whose tabIndex is " + elements[i - 1].elem.tabIndex);
+			}
 	}
 
-	export function posInSetIsDecremental(elementIds: string[]) {
+	export function posInSetIsIncremental(elementIds: string[]) {
```

Comment:
Was it testing incrementally already, but it was just the name that was incorrect?

Reply:
Yeah.... Sorry!